README.md

```diff
@@ -40,6 +40,7 @@ git clone git@github.com:uzh-rpg/rpg_vid2e.git --recursive
 
 ```bash
 conda config --add channels pytorch
+conda config --append channels conda-forge
```

Comment:
@danielgehrig18 Please also add the `conda activate vid2e` command after this line